### PR TITLE
FR-1769-add consented, contested test creation

### PR DESCRIPTION
**Pull request by vianvi**

### CODE REVIEW

**Comment by vianvi** [comment]
These testes will create consented and contested case as a solicitor user as part of PR build deployments and master.

**Comment by vianvi** [comment]
created case successfully: Screenshot 2020-08-03 at 12 06 03

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -25,7 +25,13 @@ def secrets = [
...
                 secret('userprofile-consented-judgedetails-1', 'USERPROFILE_CONSENTED_JUDGEDETAILS_1'),
                 secret('userprofile-consented-judgedetails-2', 'USERPROFILE_CONSENTED_JUDGEDETAILS_2'),
-                secret('userprofile-contested-judgedetails', 'USERPROFILE_CONTESTED_JUDGEDETAILS')
+                secret('userprofile-contested-judgedetails', 'USERPROFILE_CONTESTED_JUDGEDETAILS'),
+                secret('username-solicitor', 'USERNAME_SOLICITOR'),
+                secret('password-solicitor', 'PASSWORD_SOLICITOR'),
+                secret('username-caseworker', 'USERNAME_CASEWORKER'),
+                secret('password-caseworker', 'PASSWORD_CASEWORKER'),
+                secret('idam-client-secret', 'IDAM_CLIENT_SECRET'),
+                secret('ccd-submission-s2s-auth-secret', 'CCD_SUBMIT_S2S_SECRET')
```
Comment (shaunganley) [inline Jenkinsfile_CNP:34]: does this secret belong in s2s keyvault above or is it FR specific?

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -25,7 +25,13 @@ def secrets = [
                 secret('fixedlists-consented-judgedetails-2', 'FIXEDLISTS_CONSENTED_JUDGEDETAILS_2'),
                 secret('userprofile-consented-judgedetails-1', 'USERPROFILE_CONSENTED_JUDGEDETAILS_1'),
                 secret('userprofile-consented-judgedetails-2', 'USERPROFILE_CONSENTED_JUDGEDETAILS_2'),
-                secret('userprofile-contested-judgedetails', 'USERPROFILE_CONTESTED_JUDGEDETAILS')
+                secret('userprofile-contested-judgedetails', 'USERPROFILE_CONTESTED_JUDGEDETAILS'),
+                secret('username-solicitor', 'USERNAME_SOLICITOR'),
+                secret('password-solicitor', 'PASSWORD_SOLICITOR'),
+                secret('username-caseworker', 'USERNAME_CASEWORKER'),
+                secret('password-caseworker', 'PASSWORD_CASEWORKER'),
+                secret('idam-client-secret', 'IDAM_CLIENT_SECRET'),
```
Comment (shaunganley) [inline Jenkinsfile_CNP:33]: is this secret FR specific or does it belong in IDAM key vault?

File: Jenkinsfile_CNP (hunk 1 above)
Comment (vianvi) [reply Jenkinsfile_CNP:34]: Service secret for FR and Divorce

**Pull request merged**